src/java/com/zimbra/oauth/handlers/impl/FacebookOAuth2Handler.java

```diff
@@ -130,27 +132,31 @@
         /**
          * The authorize uri template for Facebook.
          */
-        protected static final String AUTHORIZE_URI_TEMPLATE = "https://www.facebook.com/v3.0/dialog/oauth?client_id=%s&redirect_uri=%s&response_type=%s&scope=%s";
+        protected static final String AUTHORIZE_URI_TEMPLATE =
```

Comment:
This shouldn't line break. The constants in other files are all single line regardless of length.

Reply:
Same for all the other constants, change to match.